### PR TITLE
Dev remove item Merge

### PR DESCRIPTION
**Pull request by asallh**
- Session: Added Remove item method 
- Funds: added a remove item update method 
- weight: added a remove item weight update method 
- Created exception for a product not found 
- Test file for removing products that test for items in the list + price update + weight update (runs successfully)

### CODE REVIEW

File: com.thelocalmarketplace.software.test/src/com/thelocalmarketplace/software/test/RemoveItemTests.java [hunk 1]
```diff
@@ -0,0 +1,439 @@
...
+        //Initializing weights to test for adjustment
+        Weight itemWeight = session.getWeight();
+        Mass actual = itemWeight.getExpectedWeight();
+        Mass expected = new Mass(100.0);
+        assertEquals(expected, actual);
+    	
+    }
+    
+    @Test
+    public void testRemoveDupliacateItemInDatabaseGold() {
```
Comment (kingsleyzhong) [inline com.thelocalmarketplace.software.test/src/com/thelocalmarketplace/software/test/RemoveItemTests.java:293]: nice spelling

File: com.thelocalmarketplace.software.test/src/com/thelocalmarketplace/software/test/RemoveItemTests.java [hunk 2]
```diff
@@ -0,0 +1,439 @@
...
+        
+        //Remove item
+        session.removeItem(product);
+
+    }
+    
+    
+	//remove duplicate item (update weight and price) Bronze, Silver, Gold
+    @Test
+    public void testRemoveDupliacateItemInDatabaseBronze() {
```
Comment (kingsleyzhong) [inline com.thelocalmarketplace.software.test/src/com/thelocalmarketplace/software/test/RemoveItemTests.java:236]: nice spelling v2, duplicate... dupliacate :)

**Review by kingsleyzhong — APPROVED** [review]
looks good

**Comment by asallh** [comment]
- going to rebase and fix

**Pull request closed without merging**